index.html

```diff
@@ -148,87 +175,118 @@ <h2 class="head2">Tonic</h2>
-                    <li class="dotless gh"><img src='images/gitHub.svg' alt="github icon"></li>
-                    <li class="dotless peace"><img src='images/peace.svg' alt="peace icon"></li>
+                    <li class="dotless m status"><img src='images/M.svg' alt="m icon"></li>
+                    <li class="dotless gh status"><img src='images/gitHub.svg' alt="github icon"></li>
+                    <li class="dotless peace status"><img src='images/peace.svg' alt="peace icon"></li>
+
+                    <li class="dotless gh statusd"><img src='images/gitHub.svg' alt="github icon"></li>
+                    <li class="dotless insta statusd"><img class="iinsta" src='images/i.png'><img class="ninsta"
+                        src='images/n.png' alt="instagram icon"></li>
+                    <li class="dotless peace statusd"><img src='images/peace.svg' alt="peace icon"></li>
+                    <li class="dotless twitter statusd"><img src='images/Vector.svg' alt="twitter icon"></li>
+                    <li class="dotless m statusd"><img src='images/M.svg' alt="m icon"></li>
+                    
+                    
                 </ul>
             </div>
-        </footer>
         <button class="resume">
             <div class="text-resume">Get my resume</div>
         </button>
+        </div>
+        <div class="langd">
         <ul class="lang">
             <header class="lang-txt">Languages</header>
-            <li><img src="images/Frame 9.png"></li>
-            <li><img src="images/Frame 10.png"></li>
-            <li><img src="images/Frame 11.png"></li>
+            <i class="fa-sharp fa-solid fa-angle-down dropdown3 statusd spaced"></i>
+            <li><img src="images/Frame 9.png" class="status"></li>
+            <li><img src="images/Frame 10.png" class="status"></li>
+            <li><img src="images/Frame 11.png" class="status"></li>
+            <div class="languages">
+            <li><img src="images/jsd.png" class="statusd"></li>
+            <li><img src="images/htmld.png" class="statusd"></li>
+            <li><img src="images/cssd.png" class="statusd"></li>
+            </div>
         </ul>
+        <div class="frm">
         <ul class="frameworks">
             <div class="frame">
                 <div class="frame-txt">Frameworks</div>
-                <i class="fa-solid fa-angle-right dropdown1"></i>
+                <i class="fa-solid fa-angle-right dropdown1 spaced-unique"></i>
             </div>
         </ul>
         <hr class="rule">
         <ul class="skills">
             <div class="skill">
                 <div class="skill-txt">Skills</div>
-                <i class="fa-solid fa-angle-right dropdown2"></i>
+                <i class="fa-solid fa-angle-right dropdown2 spaced-unique"></i>
             </div>
         </ul>
         <hr class="rule">
+        </div>
     </section>
-    <section class="section7">
+    <section class="section3">
+        <div class="contactd">
```

Comment:
- [ ] Kindly increase the width of the form input fields and center the elements using `flexbox` instead of using margins, because flexbox is responsive and can calculate the center of any width and it's more accurate than margins 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206857780-f30af173-bc18-487f-a275-f75f78a541c8.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206857788-9084a4ab-23bc-49f9-a421-c4488d7b7282.png)


- [ ] The form breaks at a certain screen size, the website should look great across all screen sizes so kindly fix this issue 👍 

![image](https://user-images.githubusercontent.com/50721479/206857838-3637520f-886a-4184-aac4-70ee56e9e1e3.png)